spec/acceptance/suites/default/redis_multi_instances_one_host_spec.rb

```diff
@@ -1,28 +1,44 @@
 require 'spec_helper_acceptance'
 
-describe 'redis::instance' do
+describe 'redis::instance example' do
+  instances = [6379, 6380, 6381, 6382]
   case fact('osfamily')
   when 'Debian'
     config_path = '/etc/redis'
     redis_name = 'redis-server'
   else
-    redis_name = 'redis'
     config_path = '/etc'
+    redis_name = 'redis'
   end
 
   it 'runs successfully' do
     pp = <<-EOS
-    class { 'redis':
-      default_install => false,
+    Exec {
```

Comment:
Is this exec needed? I'm not sure there's a lot of execs left in this module and I got it to work without.

Reply:
This looks like a copy'n'paste leftover, I'll remove it.